src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -109,10 +109,10 @@ public void parseClientCommands_unknownCommand_throwsParseException() {
     public void parseCountryCommands_addCountryNote() throws Exception {
         final String countryString = "SG";
         final String noteString = "is hot";
-        final String commandString = CountryNoteCommand.COMMAND_WORD + " " + PREFIX_COUNTRY + countryString
+        final String commandString = CountryNoteAddCommand.COMMAND_WORD + " " + PREFIX_COUNTRY + countryString
                 + " " + PREFIX_NOTE + noteString;
-        CountryNoteCommand command = (CountryNoteCommand) parser.parseCommand(commandString);
-        assertEquals(new CountryNoteCommand(new CountryNote(noteString, new Country(countryString))), command);
+        CountryNoteAddCommand command = (CountryNoteAddCommand) parser.parseCommand(commandString);
+        assertEquals(new CountryNoteAddCommand(new CountryNote(noteString, new Country(countryString))), command);
```

Comment:
Think this can be changed to `assertParseSuccess`

Reply:
Can't use it since `assertParseSuccess` takes in a `Parser` and `AddressBookParser` does not implement `Parser`.

Reply:
ah right my bad